docs/v0.7/en/reference/sql/range.md

```diff
@@ -23,15 +26,15 @@ INTERVAL :=  TIME_INTERVAL | ( INTERVAL expr )
    - Strings based on the `PromQL Time Durations` format (eg: `3h`, `1h30m`). Visit the [Prometheus documentation](https://prometheus.io/docs/prometheus/latest/querying/basics/#time-durations) for a more detailed description of this format.
    - `Interval` type. To use the `Interval` type, you need to carry parentheses, (for example: `(INTERVAL '1 year 3 hours 20 minutes')`). Visit [Interval](./functions.md#interval) for a more detailed description of this format.
 - `AGGR_FUNCTION(column1, column2,..) RANGE INTERVAL [FILL FILL_OPTION]` is called a Range expression.
-   - `AGGR_FUNCTION(column1, column2,..)` is an aggregate function that represents the expression that needs to be aggregated.
+   - `AGGR_FUNCTION(column1, column2,..)` is an aggregate function that represents the expression that needs to be aggregated. Aggregate functions support the use of `order by` expressions. (Note: `first_value` and `last_value` functions use timestamp as `order by` expression by default)
```

Comment:
I don't understand what NOTE means. Does that mean the type of the column specified by `order by` must be a timestamp? Does that mean that other aggregate functions can use any type of column as an order by parameter?

Reply:
The `first_value` aggregate function is used to find the first value in an aggregation group. It generally needs to be used with `order by` to specify what the first value is. In range queries, timestamps are widely used as the sorting basis. Therefore, in order to simplify the writing of SQL, the default `order by` parameter of `first_value` is timestamp, and the same for `last_value`.